test/speed_test.py

```diff
@@ -234,5 +237,13 @@ def speed_test(board_id):
         return result
 
 if __name__ == "__main__":
-    logging.basicConfig(level=logging.INFO)
-    speed_test(None)
+    parser = argparse.ArgumentParser(description='pyOCD speed test')
+    parser.add_argument('-d', '--debug', action="store_true", help='Enable debug logging')
+    args = parser.parse_args()
+    level = logging.DEBUG if args.debug else logging.INFO
+    logging.basicConfig(level=level)
+    board = pyOCD.board.mbed_board.MbedBoard.getAllConnectedBoards(close = True)[0]
+    test = SpeedTest()
+    result = [test.run(board)]
+    print result
```

Comment:
From  the commit message for this change: 

Also added printing of exception tracebacks for any exceptions that happen while running the tests.